Resources/docs/file_upload.md

```diff
@@ -8,7 +8,7 @@ Add to composer.json following lines
 
```

Comment:
I think that the best method to install component is to use composer cli. So I would rewrite this section as

```sh
composer require fsi/doctrine-extensions-bundle:^2.0
```